integration-tests/test_cdk.py

```diff
@@ -9,6 +9,8 @@
 snap_channel = os.environ.get('TEST_SNAP_CHANNEL', '1.9/stable')
 test_cloud = os.environ.get('TEST_CLOUD', '')
 bundles_csv = os.environ.get('TEST_BUNDLES', default_bundles())
+upgrade_snap_channel = os.environ.get('UPGRADE_SNAP_CHANNEL', '1.10/stable')
```

Comment:
Any reason why `1.10/stable` vs just `stable`?  The former means we'll have to remember to update this periodically.  The latter keeps current functionality.